test/java/org/chromium/net/CertificateVerificationTest.java

```diff
@@ -23,7 +24,8 @@
 public final class CertificateVerificationTest {
   static {
     AndroidJniLibrary.loadTestLibrary();
```

Comment:
I think that we should make it harder for folks to forget to call both `AndroidJniLibrary.load` and `AndroidJniLibrary.loadTestLibrary` - it's a lot of things to remember. Can we change the signature and implementation of `AndroidJniLibrary.loadTestLibrary` method so that:

1. It takes `context` as its argument
1. It calls `AndroidJniLibrary.load` internally.

That would allow us to simplify our test code. What do you think?

Reply:
AndroidJniLibrary.load() is already called in AndroidEngineImpl constructor, and any tests which create an engine which is most of the tests should have already loaded the app context and won't need to do it twice. We missed it in this test because it is unit-testing the JNI calls. So RELEASE_ASSERT in `get_class_loader()` should be sufficient to avoid similar mistakes in the future.

Reply:
OK, that makes sense (the fact that we call this method from within the Android Engine builder anyway). Let's leave it the way it is and revisit if it turns out that calling 2 methods as opposed to 1 will be confusing to people.